dockers/docker-base/etc/rsyslog.d/supervisor.conf

```diff
@@ -1,4 +1,4 @@
-module(load="imfile") #By default notify mode
+module(load="imfile" mode="inotify") # Set up notify mode
```

Comment:
Nit-pick: Suggest changing the comment to `# Ensure "inotify" mode is used`

Reply:
@jleveque Done